polyfills/Event/hashchange/config.json

```diff
@@ -11,7 +11,10 @@
 	"browsers": {
 		"ie": "6 - 7",
 		"safari": "4",
-		"samsung_mob": "*"
+		"samsung_mob": "*",
+		"chrome": "<52.0.0",
+		"firefox": "<50.0.0",
+		"ios_saf": "< 9.3.0"
```

Comment:
Same for the other hashchange records in RUM - they are false negs, not false positives.